README.md

```diff
@@ -13,31 +17,58 @@ For additional binding with DOM, browser history, etc. use [PalindromDOM](#Palin
 
 ### Installation
 
-You can install it using [bower](http://bower.io/) `bower install Palindrom` or just download from [github](https://github.com/Palindrom/Palindrom).
+#### You can install using [bower](http://bower.io/) and [NPM](http://npmjs.com/):
+
+##### Bower:
+
+```sh
+bower install Palindrom --save
+```
 
-Then add source to your head:
+Then add source to your HTML
 
 ```html
 
-<!-- include Palindrom with dependencies -->
-<script src="bower_components/fast-json-patch/src/json-patch-duplex.js"></script>
-<script src="bower_components/Palindrom/src/palindrom.js"></script>
+<!-- include Palindrom bundle -->
+<script src="bower_components/Palindrom/dist/palindrom.js"></script>
 ```
 See [Dependencies section](https://github.com/Palindrom/Palindrom#dependencies) for more details.
 
+##### NPM:
+
+```sh
+npm install palindrom --save
+```
+
+Then you can require it CommonJS or ES6/TS style:
+
+###### CommonJS:
+```js
+var Palindrom = require('palindrom');
+```
+
+###### ES6/TS:
+```js
+import Palindrom from 'palindrom'
+```
+
+Or just download it manually from [github](https://github.com/Palindrom/Palindrom).
```

Comment:
I'd add `#####` header or move it up, as it looks like belonging to `ES6/TS` section